hm11_2.py

```diff
@@ -0,0 +1,36 @@
+""" Task 2: bank deposit """
+
+
+class Bank:
+
+    """ h """
+    persent = 30
+
+    def __init__(self, summ_dep, dep_time, dep_n):
+        self.summ_dep = summ_dep
+        self.dep_time = dep_time * 12
+        self.dep_n = dep_n
+
+    def dep(self, persent):
+
+        """ j """
+        self.persent = persent
+        monthly_persent = persent/100/12
+        persent_total = self.summ_dep * (1 + monthly_persent) ** self.dep_time
+        return persent_total
```

Comment:
round final amount to 2 digits after `.`

Reply:
Done